src/views/base.njk

```diff
@@ -2,46 +2,24 @@
 <html lang="en">
 <head>
     <title>the federation - a statistics hub</title>
-    <meta charset="UTF-8">
-    <meta property="og:title" content="the federation - a statistics hub">
-    <meta property="og:type" content="website">
-    <meta property="og:url" content="https://the-federation.info/">
-    <meta property="og:description" content="Podlist and statistics for The Federation (diaspora*, Friendica, Hubzilla).">
-    <meta property="og:image" content="https://the-federation.info/images/the-federation.png">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <link rel="stylesheet" href="/vendor/css/footable.core.min.css">
-    <link rel="stylesheet" href="/vendor/css/footable.standalone.min.css">
-    <link rel="stylesheet" href="/vendor/css/hubinfo.min.css">
-
-    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css">
-    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap-theme.min.css">
-
-    <link rel="stylesheet" href="/vendor/css/rickshaw.min.css">
-
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```

Comment:
This is why the graphs or the info page don't work - used to fetch these resources (jquery, bootstrap, d3, floathead) from a CDN (evil, I KNOW) - now they fetched locally but the files are not there.

Maybe add the relevant files instead of fixing the references? Would be nicer if they were local while we still need them.